sdk/lib/core/map.dart

```diff
@@ -278,13 +355,14 @@ abstract class Map<K, V> {
   /// Otherwise calls [ifAbsent] to get a new value, associates [key] to
   /// that value, and then returns the new value.
   /// ```dart
-  /// Map<String, int> scores = {'Bob': 36};
-  /// for (var key in ['Bob', 'Rohan', 'Sophena']) {
+  /// final scores = {'Bob': 36};
+  /// for (final key in ['Bob', 'Rohan', 'Sophena']) {
   ///   scores.putIfAbsent(key, () => key.length);
   /// }
-  /// scores['Bob'];      // 36
-  /// scores['Rohan'];    //  5
-  /// scores['Sophena'];  //  7
+  /// print(scores); // {Bob: 36, Rohan: 5, Sophena: 7}
+  /// // Return value is current value and no update done if key exists on map
```

Comment:
```suggestion
  /// // If the key already exists, the current value is returned, and the Map isn't modified.
```

Reply:
Also wondering if this should be a separate example - or if not, at least separate with a new line